samples/Mvc.Client/Properties/launchSettings.json

```diff
@@ -24,4 +24,4 @@
       }
     }
   }
-}
```

Comment:
This hasn't been resolved.

Reply:
To aid reviewers, please don't resolve their comments yourself. It's easier to see what was said and whether they've been auctioned if they're not collapsed after changes are pushed. The reviewer themselves can then resolve when they're satisfied a point has been actioned.

Reply:
The result is same you have minus and after plus. This is because I use vs for mac and something it works differently than vs for windows.

Reply:
Whether or not you use an editor that obeys [EditorConfig rules](https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers/blob/2be2d28c45e3b619c9e5017acc2ed4afb084a62a/.editorconfig#L13), this change just adds churn when it could just not have been included in the commit in the first place when you staged it.